README.md

```diff
@@ -14,8 +14,17 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 <!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
 
-1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+1. Summarize proteome [`BUSCO`](https://busco.ezlab.org/) completeness at both taxon-specific level, and compared to Eukaryota
```

Comment:
Okay, i don't know how to respond to your main comment abov, but you've definitely made it clear to me that I should move the modified nf-core modules to local - there are several.

As for the R packages installed in modules - I agree completely, I just need to make a docker image for those that includes what I need. The package installation approach was something dumb and hacky I implemented before I knew how to create my own docker images. 

Yeah, I probably need to rework some of this to get it into shape to pull data from S3 - some of my implementation here stems from my not know what I was doing early on in the development of the workflow. 

The "test" dataset is a subset of samples that is used to assess MCL inflation parameters for clustering into orthogroups, and the "full" dataset is the one we actually want to do all inferences on. I needed them to be in their own directories since Orthofinder creates its own directory structure wherever the data is stored. So, I sadly had to work within those bounds. 

And I agree 100% that this is the wrong way to do a PR and I apologize for that! Bite sized chunks from here on out!!

~~~~~~~~~~~~~~~~~~

As for the question about BUSCO, I suppose it could be broken out entirely, but at least for my use case I did a fair amount of filtering of assemblies prior to this point, and so figured it would be nice to include. I had also considered using the completeness result to filter individuals somewhat, but I may still build a different module for this purpose. 

Ultimately I think I included this because it's something that I would like done, but maybe doesn't actually need to be included here.